### DISCUSSION TITLE
How to correctly name/cite head segmentations

### DISCUSSION THREAD

**Discussion by dyollb** [simnibs/simnibs#405] [7.Creating head models] [answered]
I am comparing segmentation accuracy between the old headreco and newer charm "variants".
I noticed there are different segmentations provided by SimNIBS >=v4 in the folders
- m2m_SomeSubject
  - label_prep
  - segmentation
  
1. Do you have a convention for naming the labelfields in `label_prep` and `segmentation`. 
2. What is their respective intended use? e.g.,
  - Do users typically run simulations with meshes based on `label_prep`? 
  - Is the labelfield in `segmentation` mainly intended for identifying targets?
 
 Should I cite different papers, depending which segmentation (`label_prep` vs `segmentation`) I use?

**Comment by oulap** [comment, marked as answer]
Hi Bryn,

The segmentation folder contains a full neuroanatomical segmentation plus all the head tissues. The brain labels are the same as in FreeSurfer, and the extra-cerebral tissues are the ones described in the charm paper. 

The label_prep folder is used to prepare the aforementioned segmentation for meshing. First the segmentation is upsampled to roughly twice the resolution of the input data, next labels in the "full" segmentation are combined into tissues (WM, GM, CSF, etc), and finally we use some morphological operations to clean out the upsampled tissue map. That map is then fed into the meshing to produce the head model.

So the mesh is always based on the tissue_upsampled.nii.gz file inside label_prep, which is based on the full segmentation inside the segmentation folder. The simulations are run on meshes constructed on label_prep. The full segmentation could be used for targeting but the main purpose of it is to construct meshes, although with an extra upsampling and label merging step in between.

You can just cite the charm paper for both, but if you're specifically doing something with the stuff inside the segmentation folder it's probably good to also cite this paper:

https://www.sciencedirect.com/science/article/pii/S1053811916304724

Hope this helps!

**Comment by dyollb** [comment]
Thanks, this clarifies most of my questions. Regarding naming, can I use the following?
- "charm" or "charm (default)": merged/smoothed segmentation used for meshing
- "charm+freesurfacer" or "charm (full)": full segmentation

Thanks

**Reply by oulap** [reply]
Yep, sounds good to me.
